packages/react/src/components/Popover/Popover-story.js

```diff
@@ -0,0 +1,442 @@
+import Button from '../Button';
+import mdx from './Popover.mdx';
+import { useOutsideClick } from '../../internal/useOutsideClick';
+
+export default {
+  title: 'unstable_Popover',
+  component: Popover,
+  subcomponents: {
+    PopoverContent,
+  },
+  decorators: [withKnobs],
+  parameters: {
+    docs: {
+      page: mdx,
+    },
+  },
+  includeStories: ['Playground'],
+};
+
+export const Default = () => {
+  function PopoverDemo() {
+    const [align, setAlign] = React.useState('top');
+    const [open, setOpen] = React.useState(true);
+    const choices = [
+      'top',
+      'top-left',
+      'top-right',
+
+      'bottom',
+      'bottom-left',
+      'bottom-right',
+
+      'left',
+      'left-bottom',
+      'left-top',
+
+      'right',
+      'right-bottom',
+      'right-top',
+    ];
+
+    return (
+      <>
+        <section>
+          <h2>Caret Position</h2>
+          <div
+            style={{
+              display: 'grid',
+              gridTemplateColumns: 'repeat(3, auto)',
+              gridGap: 8,
+            }}>
+            {choices.map((choice) => {
+              return (
+                <label key={choice}>
+                  <input
+                    type="radio"
+                    name="align"
+                    value={choice}
+                    checked={choice === align}
+                    onChange={() => {
+                      setAlign(choice);
+                    }}
+                  />
+                  {choice}
+                </label>
+              );
+            })}
+          </div>
+        </section>
+        <section>
+          <h2>Popover Visibility</h2>
+          <div>
+            <label>
+              <input
+                type="checkbox"
+                value={open}
+                checked={open}
+                onChange={() => {
+                  setOpen(!open);
+                }}
+              />
+              Open
+            </label>
+          </div>
+        </section>
+        <div style={{ marginTop: '2rem' }}>
+          <Popover open={open} align={align} relative>
+            <PopoverContent className="p-3">Hello</PopoverContent>
+          </Popover>
+        </div>
+      </>
+    );
+  }
+
+  return <PopoverDemo />;
+};
+
+export const Playground = () => {
+  const options = [
+    'top',
+    'top-left',
+    'top-right',
+
+    'bottom',
+    'bottom-left',
+    'bottom-right',
+
+    'left',
+    'left-bottom',
+    'left-top',
+
+    'right',
+    'right-bottom',
+    'right-top',
+  ];
+
+  return (
+    <Popover
+      align={select('Specify the caret alignment', options, 'top')}
+      caret={boolean('Specify whether the caret should be visible', true)}
+      highContrast={boolean(
+        'Specify whether the high-contrast variant should render',
+        false
+      )}
+      light={boolean('Specify whether the light variant should render', false)}
+      open={boolean('Specify whether the popover is open or closed', true)}>
```

Comment:
we typically include the prop names in the label right?

```suggestion
      align={select('Specify the caret alignment (align)', options, 'top')}
      caret={boolean('Specify whether the caret should be visible (caret)', true)}
      highContrast={boolean(
        'Specify whether the high-contrast variant should render (highContrast)',
        false
      )}
      light={boolean('Specify whether the light variant should render (light)', false)}
      open={boolean('Specify whether the popover is open or closed (open)', true)}>
```